backend/services/auth.js

```diff
@@ -0,0 +1,20 @@
+const User = require('../models/user');
+const authHelper = require('../helpers/auth');
+
```

Comment:
We may implement service class like authService = {
signUp: async (params) =>{},
anotherFunction: async (params)=>{}
}

Reply:
I find the current way better than your propasal because of one less level of indentation. Also, we may use a linter to determine the style guides that we should follow. I will create an issue for that